_posts/2021-03-27-RIP-pullmoll.markdown

```diff
@@ -0,0 +1,28 @@
+---
+layout: post
+title: "RIP pullmoll"
+comments: true
```

Comment:
What does this do?

Also, remember to squash the commits together before merging.